js/index.js

```diff
@@ -16,9 +11,44 @@ const skills = ['Skill 1', 'Skill 2', 'Skill 3', 'Skill 4'];
 const skillsSection = document.querySelector('#skills');
 const skillsList = skillsSection.querySelector('ul');
 for (item of skills) {
-    console.log(item);
     const skill = document.createElement('li');
     skill.textContent = item;
     skillsList.appendChild(skill);
 
 }
+
+// handle message form submit
+const messageForm = document.getElementsByName('leave_message')[0]; // is this the best way to do it?
+messageForm.addEventListener('submit', (event) => {
+    
+    event.preventDefault(); // event or event.target? why this is needed at all?
+
+    // local const
+    const name = event.target.name;
+    const email = event.target.email;
+    const message = event.target.message;
+
+    // select elements
+    const messageSection = document.getElementById('messages');
+    const messageList = messageSection.querySelector('ul');
+
+    // remove button
+    const removeButton = document.createElement('button');
+    removeButton.textContent = "remove";
+    removeButton.type = "button";
+    removeButton.addEventListener('click', (event) => {
+        event.target.parentNode.remove();
+    });
+
+    // new message
+    const newMessage = document.createElement('li');    
+    newMessage.insertAdjacentHTML(
+        'afterbegin',
+        `<a href="mailto:${email.value}">${name.value}</a> wrote <span>${message.value} </span>`
+    )
+
+    newMessage.append(removeButton);
+    messageList.appendChild(newMessage);
+   
+    event.target.reset(); // why event target in this case?
```

Comment:
The `event` is the form submission. The event happened on an element which we can see who it was by `event.target` 

Because we want to reset the form - and the submission happened on the form - we can say reset the events' target... giving us `event.target.reset()` 

This is something I had to look up though - so don't worry too much - things like this you'll probably google later on and copy paste.